client/app/components/packages/rwcds-form/with-action-no-action.hbs

```diff
@@ -51,9 +51,9 @@
             (hash label='Yes' code=true)
             (hash label='No' code=false)
           }}
-          @onChange={{fn (mut @form.saveableChanges.dcpDescribethenoactionscenario) ""}} as |value|
```

Comment:
so this was previously a mistake?

Reply:
It wasn't necessarily a mistake but it did only take into account ONE "reset" of a field when they're technically should have been TWO, since two fields were conditional. BUT now that only one field depends on this condition, I just needed to update it to reflect that one.